atm.js

```diff
@@ -1,21 +1,123 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
 
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
+//select / create elements to work with
+console.log('ready');
+var checkingBox = $('#check-input');
+var checkingBoxVal = parseInt(checkingBox.val());
+var savingsBox = $('#savings-input');
+var savingsBoxVal = savingsBox.val();
+var checkBalanceDisplay = $('#checking-display');
+var savingsDisplay = $('#savings-display');
+  //checking buttons
+var checkDBtn = $('#checkbtn-d');
+var checkWBtn = $('#checkbtn-w');
+  //savings buttons
+var savingsDBtn = $('#savingsbtn-d');
+var savingsWBtn = $('#savingsbtn-w');
 
-}
 
-function checkingWithdrawal(amount){
+//start w/nothing --> show red
+checkBalanceDisplay.addClass('zero');
+savingsDisplay.addClass('zero');
 
-}
+  //make bank account constructor
+function Accnt() {
+    this.totalBal = 0;
+    this.deposit = function(dep_amnt) {
+      return this.totalBal += dep_amnt;
+    };
+    this.withdrawal = function(withdraw_amnt) {
+      return this.totalBal = this.totalBal - withdraw_amnt;
+    };
+  }
+  //make bank accounts, one for checking, one for savings
+var checkingAccnt = new Accnt();
+var savingsAccnt = new Accnt();
 
-function savingsDeposit(amount){
+//event handlers
+  //checking account
+    //on checking deposit click, deposit
+checkDBtn.click(function() {
+    checkingAccnt.deposit(parseInt(checkingBox.val()));
```

Comment:
I might place these click events into separate function, for example: ```
function addClickEventToCheck (){
    checkDBtn.on("click", function() {
    })
 }```

It's not necessary, but might be useful if you want to stop and restart a click event again in the future